src/core/relay/group/type.ts

```diff
@@ -1,4 +1,14 @@
+import { WellKnownEventKind } from 'core/nostr/type';
 import { Relay } from '../type';
 
-export type RelayGroupId = string;
-export type RelayGroupMap = Map<RelayGroupId, Relay[]>;
+export interface RelayGroup {
+  id: string;
+  title: string;
+  description?: string;
+  relays: Relay[];
+  timestamp: number;
+  kind?: WellKnownEventKind;
+  changed?: boolean;
```

Comment:
what does this field `changed` do?

Reply:
The “changed” field is not present by default, it is only added when the relay group is modified locally. 
This field indicates that the relay group has been modified and the sync button will only be displayed when `changed = true` (Will delete this field after local modifications to the relay group are published in the NIP-51 relay set event).